### PR TITLE
Digimon Showdown

### PR DESCRIPTION
**Pull request by DeathlyPlays**
The Digimon Project.  We have one thing we really wanted to add being sprites.... but being side servers we really couldn't edit the client....  We were going to use regular Pokemon sprites as you may notice by the `num: (insert number here`, but we couldn't inherit it properly like how we wanted.  Nevertheless here it is, right before April Fool's....

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
It is guaranteed to be stable, we've tested.

**Comment by Zarel** [comment]
This actually looks really good, but we don't have time to get sprites for this. Let's save this for next year.

**Comment by DeathlyPlays** [comment]
Spriting issue is fixed and stuff :P

**Comment by DeathlyPlays** [comment]
We have sprites (tbh they're just Pokemon sprites) added now

**Comment by AshleyPikachu** [comment]
Hi Zarel, I'm Ashley the Pikachu. I'm the founder of the project. While not a major coder, I could easily set you up with some sprites if you would like. I would just need to know the size limits.

**Comment by HoeenCoder** [comment]
If your interested in looking at the final product (with regular sprites) it was merged into the master branch of my side server (spacialgaze) here https://github.com/HoeenCoder/SpacialGaze/pull/172 its also playable on spacialgaze.

**Comment by Stellation** [comment]
There are SOME bugs with items, but besides that its good and ready to play flawlessy

**Comment by AshleyPikachu** [comment]
and a few minor animation bugs we can fix later but yea its pretty stable. :)

**Comment by Zarel** [comment]
Normal sprites are 96x96 PNG files. We also support animated GIF sprites, which can be any size.

**Comment by AshleyPikachu** [comment]
Okay thanks for letting me know. So with that being said If I find the sprites will you set them officially? Also this is just a quickly made example I made as png. I'm not sure if I will be doing Gifs or not yet. http://imgur.com/a/fElpD

**Comment by Zarel** [comment]
Yes, we would.

**Comment by AshleyPikachu** [comment]
Okay, I will the team know tomorrow. :) That being said were undecided on 2D or 3D yet due to having to find the correct sprites. Anyways I will keep you updated.

**Comment by DeathlyPlays** [comment]
Haha, this is the dev team! ^
Devs:
Ashley the Pikachu (Head Researcher, Began the project, Attack Manual, Type Chart Manual, Music Selection and Music HTML, Sprite Selection), Insist (Head Developer), AlphaStorm (Animations), Stellation (Assisted other developers), HoeenHero (Assisted with Mechanics)

**Comment by DeathlyPlays** [comment]
Also, if you really want to when we get the sprites up and everything on the PS client, we could have this as a meta (OM of the month or just a meta that stays here)

**Comment by AshleyPikachu** [comment]
Oh by the way while i do sprite research today. I know you have to add new slots for these. Eventually we would like to add more digimon to make the meta up to date to modern standards. Should we just get any future sprites in advance now or do it when the time comes?

**Comment by Stellation** [comment]
@DeathlyPlays Here's the thing, we can't have it as an OMoTM, cause it's not an other meta. It would be a seasonal metagame like super staff bros melee

**Comment by AustinXII** [comment]
I think this would be much better just as an afd joke

**Comment by Stellation** [comment]
Maybe for next year we can use this, or maybe as a seasonal for April 2018, idk

**Comment by AshleyPikachu** [comment]
I'm just happy we get real sprites :D, by the way I also made graphics for the remaining type chart. Can we include these too if that's alright? http://imgur.com/a/kre6p Fire and Ice don't have a graphic because they don't need it since it already exist.

**Comment by Stellation** [comment]
that DOES sound like a good idea in theory. but in practice, god knows how that can be coded in, we may have a better chance of that working if the custom species with sprites happen to get in. @AshleyPikachu

**Comment by DeathlyPlays** [comment]
@Stellation via CSS that can be coded in

**Comment by Zarel** [comment]
We don't really use those, though. The type icons we use look like
image

**Comment by HoeenCoder** [comment]
@AshleyPikachu if you mean the move buttons, thats done with CSS not images

**Comment by Stellation** [comment]
True, we only have up to the css of Pokemon showdown to work with.

**Comment by Stellation** [comment]
Due to a new decision by the dev team: the format is now 6v6 triples

**Comment by DeathlyPlays** [comment]
^ Also eh I thought the CSS was coded as a background image for the types >_< my bad

**Comment by DeathlyPlays** [comment]
So we need to make a pull request to the client and @Zarel can you accept the client update for the Digimon sprites and stuff before we release it to main, it would make it more fun when we test it on SpacialGaze or Exiled.

**Comment by HoeenCoder** [comment]
Sprites aren't on the client repo, those would be added manually.

**Comment by AshleyPikachu** [comment]
Alright, I found a sprite set I like, they are in black and white so I have to put them in color so its gonna take awhile before there done and they will be Gifs as well. 

The bottom URL is what has been done so far. The top Botamon will be your opponents Digimon and the one with the side view will be your Digimon. 

After that its a before and after shot of Agumon and The bottom image is just a rough concept of what that is suppose to look like and is not to scale. 
http://imgur.com/a/Ltdpf

**Comment by Stellation** [comment]
Full potential bug is fixed as seen in this replay: http://replay.pokemonshowdown.com/spacialgaze-gen7digimonshowdown-7991

**Comment by AshleyPikachu** [comment]
Alright turns out I do have a update on the sprites. I have decided to go with the Dot Versions of digimon for sprites. What this means is there V-pet Graphics, but I can animate a lot faster and we only need 1 set of sprites 2 at the most if its a different file for them being flipped. My only Question is the default size to small which i can make them bigger no issues. Also would you like use to gather custom backgrounds for this meta or no? Here is an example of a completed Agumon sprite at its normal size and it 6 of them on the battle field made larger. http://imgur.com/a/4GPeD

**Comment by Stellation** [comment]
A recent decision by our Dev team decided that abilities should be nerfed. This is to help balancing certain Digimon and moves that are too powerful with the boost

**Comment by DeathlyPlays** [comment]
@Zarel, @AshleyPikachu has questions about if you want the cries, and how you want us to send you the background.

**Comment by DeathlyPlays** [comment]
@HoeenCoder you can continue the project.

**Comment by AshleyPikachu** [comment]
@Zarel I just finished the sprites last night, I need to find a gif creator that doesn't lose quality. Anyways Ive made a test version of Agumon http://i.imgur.com/B10R8W1.gif hes 56 by 56 pixels and all dot digimon are the same size. Hes about the size of a charmeleon. Is this size format alright or does it need to be in the middle of white space like the BW sprites?

**Comment by Zarel** [comment]
Sure, it's fine.

**Comment by AshleyPikachu** [comment]
@Zarel awesome, I wasn't sure if it needed to be in the center of white space or not. Good to know its not needed. Makes animating a bit faster on my part.

**Comment by AshleyPikachu** [comment]
@Zarel okay I found a gif creator i like, This will be the final size for all sprites, do let me know if this is okay. still 56 by 56 pixels. http://imgur.com/a/w1MFT I put charmelon in there as a size refrence and to show how these sprites don't have any extra white space like charmelon does.

**Comment by AshleyPikachu** [comment]
@Zarel one more thing i forgot to mention our roster is now 164 monsters compared to 62 when we got our first build working. We still have balancing and content update to add but that is the final size. :)

**Comment by Zarel** [comment]
Works.

**Comment by AshleyPikachu** [comment]
@Zarel Great, I will let you know when im done asap :)

That being said, i know you said we can have custom sounds. What kind should i get? I know pokemon crys, is there anything else i should grab. Also should i get something for the pokeball throw sprite?

**Comment by AshleyPikachu** [comment]
@Zarel Okay, I guess i'll just grab what i think is good and let me know if it can be used.

**Comment by AshleyPikachu** [comment]
@Zarel Ive made several digimon now and I thought it be best to send you some samples. Let me know if there any any problems with them. That will be exactly how there sent to you when we are ready. http://imgur.com/a/w1MFT

**Comment by Zarel** [comment]
Looks usable.

**Comment by AshleyPikachu** [comment]
@Zarel awesome, i was worried the lack of white space would be a issue good to know. Thanks Zarel.

**Comment by AshleyPikachu** [comment]
@Zarel the Sprites for the monsters are done :), now I know they need smicon sprites which is no problem but I was wondering how you would like them. I could rip them into single files or you can just pick the one you like out of the sprite source file since its the size. Let me know what you would like to do.

**Comment by Zarel** [comment]
is everything else also done?

**Comment by AshleyPikachu** [comment]
@Zarel not yet. We are still working on polish for the code. But we are getting there. :)

**Comment by Zarel** [comment]
Okay, I'll wait until you're done before I upload. Ideally you just put them in the same directory layout as the other sprites are, and send them to me as a ZIP (presumably attached to the pull request).

**Comment by DeathlyPlays** [comment]
Trying to make this meta as easily playable as possible

**Comment by AshleyPikachu** [comment]
@Zarel I assume when its on client we can have our custom attacks have colors assosiated with them correct? Is that done with a hex color or do I have to make some png and send them?

**Comment by Zarel** [comment]
Can't you just use a Pokémon type with that color?

**Comment by AshleyPikachu** [comment]
@Zarel if thats possible sure.

**Comment by Lycanium-Z** [comment]
Hi its Stellation! Changed names in the process.

Anyways heres is what i could envision the colors: Fire = Flame, Water = Aquatic, Flying=Air, Nature=Grass, Battle = Fighting, Mech = Steel, Filth=Rock, Evil=Dark, Light= Electric

File: config/formats.js [hunk 1]
```diff
@@ -1356,4 +1356,21 @@ exports.Formats = [
...
+		},
+		desc: [
+			"You may have thought this was Pok&eacute;mon Showdown, but I must reassure you that you were mistaken, welcome to <b>DIGIMON SHOWDOWN!!!</b>",
+			"Ahem, but in all seriousness, the following developers listed below created Digimon in Pok&eacute;mon Showdown, so why not just try it out :D",
```
Comment (AustinXII) [inline config/formats.js]: I may just be being too harsh here but could this be changed so it Isn’t so cringy to read?

Comment (Lycanium-Z) [reply config/formats.js]: We're revamping everything, thatll probably changed as well.

**Comment by AshleyPikachu** [comment]
@zarel devleopment is going smooth we are almost done with all the paper work and about to begin final coding. Also Pokemon X Digimon will be a meta game option too. We have already planned out the type chart and stuff.

**Comment by AshleyPikachu** [comment]
This is the type chart @Zarel https://cdn.discordapp.com/attachments/345314312010858507/367351142054625281/Digimon_Showdown_Type_Chart.png

**Comment by AshleyPikachu** [comment]
@Zarel I can do custom cries right?

**Comment by Zarel** [comment]
...I would rather not but it's technically possible.

**Comment by AshleyPikachu** [comment]
@zarel okay no worries then can I assign them pokemon ones then? Also is anything else off limits I got some custom backgrounds made.

**Comment by AshleyPikachu** [comment]
Btw I would like to say I have the cries but I understand you not wanting to use them. So no harm done I just found them all yesterday.

**Comment by Zarel** [comment]
It'd definitely be easier to give them custom cries than assign them pokemon ones

**Comment by AshleyPikachu** [comment]
@Zarel so I can go forward and get them ready? Is there any sound format you prefer?

**Comment by AshleyPikachu** [comment]
If not no biggy just let me know, them being mute wont be the end of the world.

**Comment by Zarel** [comment]
The files should be in the same format as current sounds are. `[speciesid].[ext]`.

**Comment by Zarel** [comment]
http://play.pokemonshowdown.com/audio/cries/

**Comment by AshleyPikachu** [comment]
Perfect Mp3 and Ogg easy, I'll have that ready to go before launch, Should the custom backgrounds have special names too?

**Comment by AshleyPikachu** [comment]
Also thanks @Zarel :)

**Comment by AshleyPikachu** [comment]
@Zarel Here are our SFX, I figure might as well give them to you now. https://1drv.ms/u/s!AvoD6RnUzzMvgmOW1S9u6sHIX8-b Its both Ogg and Mp3

**Comment by AshleyPikachu** [comment]
Thanks again for letting us have sfx

**Comment by AshleyPikachu** [comment]
@Zarel it has come to my attention lilymons sound file has its japanese name of "Lilymon" in the US its spelled like "Lillymon" I have corrected this on my files but it is incorrect in the audio files I sent you back in Nov. its a easy file to adjust or I can just resend you everything in one big bundle when we are done. What would you like to do.

**Comment by Zarel** [comment]
I haven't actually downloaded it; I was planning on waiting for it in one big bundle when you're done.

**Comment by AshleyPikachu** [comment]
okay sounds good to me.

**Comment by AshleyPikachu** [comment]
Hi again, I was wondering what would you like the gif files to be called also do you need single frame png versions of them for example agumon is called "Agumon P.gif" and"Agumon O.gif" P standing for player and O standing for opponet.

**Comment by KrisOnline** [comment]
@AshleyPikachu This is just a suggestion, but I would do it like this:

Make two folders: `digimon` and `digimon-back`.

Put the files labeled like `Agumon P.gif` in the `digimon-back` folder, remove the ` P` from the name, and lowercase everything (ie `agumon.gif` in the `digimon-back` folder: `digimon-back/agumon.gif`).

Put the files labeled like `Agumon O.gif` in the `digimon` folder, remove the ` O` from the name, and lowercase everything (ie `agumon.gif` in the `digimon` folder: `digimon/agumon.gif`).

I haven't looked too closely at the code, so I'm not sure if there are any formes. If there are, I'd do the same protocol as above, except I'd replace the spaces with underscores (or just flat out remove the spaces), so something like `Digimon One.gif` would become `digimon_one.gif` or `digimonone.gif`. This is how PS currently handles sprites, as seen [here](http://play.pokemonshowdown.com/sprites/).

**Comment by AshleyPikachu** [comment]
Okay what should I do about the pngs, they are 56 by 56 pixels, should I make the file bigger without making the monster bigger?

**Comment by KrisOnline** [comment]
yes

**Comment by AshleyPikachu** [comment]
okay will do thanks for letting me know.

**Comment by AshleyPikachu** [comment]
@KrisOnline 5 of the digimon have spaces in there name cherubimon (good), cherubimon (evil), metalgreymon (vaccine), metalgreymon (virus) and meicrackmon vm. is this okay?

**Comment by AshleyPikachu** [comment]
by the vm stands for Vicious mode, its a seprate version of Meicrackmon, think of it like Mewoth and Meowth Alola.

**Comment by AshleyPikachu** [comment]
The files are
metalgreymon (vaccine).gif
metalgreymon (virus).gif
meicrackmon (vicious mode).gif
cherubimon (good).gif
cherubimon (evil).gif

**Comment by AshleyPikachu** [comment]
Also if those are fine, the gifs are done what file format do i do for the pngs, I see 2 versions in Xy pokemon.png and pokemon-f.png

**Comment by KrisOnline** [comment]
remove the parentheses and replace the spaces with hyphens. the space in "vicious mode" should just be removed and not replaced (`meicrackmon-viciousmode.gif`)

pokemon.png and pokemon-f.png mean a Pokemon's male appearance (pokemon.png) and female appearance (pokemon-f.png). If there are no gender specific appearance changes, there is no need for two files.

**Comment by AshleyPikachu** [comment]
okay thanks, yea there all genderless

**Comment by AshleyPikachu** [comment]
Okay got the file names fixed, thanks for the help. Now Cherubimon and Metalgreymon are kind of like Aolan versions same name just ones virus and ones vaccine is it possible to hide the name tag like with alolans.

**Comment by AshleyPikachu** [comment]
Also what do you guys plan to do with the smicons-sheet.png, should I just send you all 172 mini sprites in a bundle and you guys stick them in the right spot?

**Comment by AshleyPikachu** [comment]
@KrisOnline i found this directory, I can make some icons for the Digimon types want me to?

**Comment by AshleyPikachu** [comment]
Nevermind I'll just make them its gonna be super easy.

**Comment by AshleyPikachu** [comment]
@KrisOnline and @Zarel i got type icons finished. https://i.imgur.com/zEiUV4V.png

**Comment by AshleyPikachu** [comment]
okay i got the sprites ready, @KrisOnline what do i do about the tiny icon sprites?

**Comment by AshleyPikachu** [comment]
https://play.pokemonshowdown.com/sprites/xyicons-sheet.png this sheet

**Comment by AshleyPikachu** [comment]
@Zarel the sprite sheet is done how does it look? https://cdn.discordapp.com/attachments/355121227314102274/397982757369806848/digimonicons-sheet.png

**Comment by Zarel** [comment]
Looks workable!

**Comment by AshleyPikachu** [comment]
awesome :D

**Comment by AshleyPikachu** [comment]
Hey @Zarel I was wondering, how should we approach teaching people how to play the game. Since it was custom types and new moves. I'm not sure how you would like like approached since we don't have a team builder.

**Comment by AustinXII** [comment]
Maybe you guys could make a post on smogon when this actually gets implemented and have a link to the rules at the start of every battle?

**Comment by Zarel** [comment]
I had an idea and was about to post it but I realized Austin basically just posted exactly the idea I had.

**Comment by AshleyPikachu** [comment]
I like that idea, how would it work I don't know a lot about how smogon structures this sorta thing. Could I get an example?

**Comment by HoeenCoder** [comment]
@AshleyPikachu I assume @AustinXII means a thread in the pokemon showdown forum on smogon.

**Comment by AshleyPikachu** [comment]
@Zarel is that correct a thread on the smogon fourm?

**Comment by AustinXII** [comment]
Putting it on smogon would probably be best.

**Comment by AshleyPikachu** [comment]
okay @AustinXII could you send me a link of one already done by someone as a example please.

**Comment by AustinXII** [comment]
Probably something like this, http://www.smogon.com/forums/threads/inheritance-om-of-the-month-blacephalon-banned-check-post-597.3592844/

File: config/formats.js [hunk 2]
```diff
@@ -1356,4 +1356,21 @@ exports.Formats = [
...
...tGOvH2m6TNHfGwEVBnSFYmJnQ5irfZG5PTm2Q_cvEZSVJOjco65nxgpmgxkh9u6-Z2-67FqIWD4oMfceTWGOqlL0n9zDM8OrjYKBhRI3FqA8vZq_eAc1QSWIRCdWDa21vklS4XA?width=124&amp;height=28&amp;cropmode=none" alt="Digimon Frontier: With the WIll BGM" width="124" height="28" /></a><a title="Digimon Adventure: Shuugeki Soshite..." href="https://www.youtube.com/watch?v=66iMd-lpC9E" target="_blank"><img src="https://x5zilq.bn1303.livefilestore.com/y3mIjynkF0J_iQQ0IeK0jCR1BLEIpgDkRRGUdYSyOOeNyFcaWCkI5W6LrQamjbqgibv203Ss824bh24kF_gBWNhxOBkjg_rdimfGBsN_6a1k64KydPJRTTrZdA6naRzTBOoxg59qQVtt6I0vQ3d5iAfLRoM_JirnCb7jLX-Ok93YL8?width=124&amp;height=28&amp;cropmode=none" alt="Digimon Adventure: Shuugeki! Soshite..." width="124" height="28" /></a><a title="Digimon Xros Wars: Battle In The Digital World" href="https://www.youtube.com/watch?v=xngD8cDZLRQ" target="_blank"><img src="https://x5x9og.bn1303.livefilestore.com/y3mQAOGzN350mT9JOODjaIzJQ4zcLmZ1l4T_5PeAeN-m4zSj4e2tODh563uxlJh8ZflX4HJOOCtXAi5Bchow8eSOIwPcYEWToRrlD-DKcJIB7HxFdKP6zOZceolZjxKSbzk8R9rfKaSqG7y2vn7MfWJ8AQHHipXVt_cEf0vULk-d98?width=124&amp;height=28&amp;cropmode=none" alt="Digimon Xros Wars: Battle In the Digital World" width="124" height="28" /></a></center>');
```
Comment (AustinXII) [inline config/formats.js]: Is this necessary?

**Comment by AshleyPikachu** [comment]
Okay I can do that. :)

File: config/formats.js (hunk 2 above)
Comment (acz13) [reply config/formats.js]: I don't even think those onedrive files are accesible (or whatever they are)

**Comment by AshleyPikachu** [comment]
@Zarel it appears there was a bit of a hicup on the smogon fourms today. I posted the Digimon Showdown article under om. It said it was private until approve by someone and well it got approved early so a few users saw it.  I would not worry about any harm being done since most of the digimon members know about it.  Anyways sorry for the hiccup, any mention of it being for april fools day has been removed from the post. That being said the post isn't done yet.

**Comment by Zarel** [comment]
You should probably work on it somewhere else. Heck, it could be a `.md` file in your repo.

Would probably be easier as a GitHub issue in your repo, though. Then you can attach images a lot more easily.

**Comment by Zarel** [comment]
A GitHub Gist might also work. I think those let you attach images, but I've never actually tried.

**Comment by KrisOnline** [comment]
It was closed because you posted it to the OM forum without approval from one of the mods. Not to mention I don't think it should be posted on Smogon at all until april fools

**Comment by AustinXII** [comment]
Kris mimics my feels exactly. You shouldn’t spoil this until April fools. Also I would use the actual PS section for when you do post a thread unless you just want a .md file like zarel suggested.

**Comment by AshleyPikachu** [comment]
My aplogies for the mix up, I didn't realize it was going to go public that quickly. Ive made a Libre Office back up so the fourm post can be removed.

**Comment by AshleyPikachu** [comment]
I see the fourm has been allowed to stay online, Thank you and once again sorry for the mix up.

**Comment by KrisXV** [comment]
It should still be deleted... keeping that thread up will only spoil the AFD joke.

**Comment by AustinXII** [comment]
^ to clear confusion, PS and smogon are different websites with different members of authority. Zarel is the only Dev who actually has a “say” on smogon and all of it’s threads. Coincidentally there are some PS devs who mod OM but yeah please remove the spoiler, it isn’t our intention.

**Comment by AshleyPikachu** [comment]
Okay, How do I delete it?

**Comment by AustinXII** [comment]
Click the button beside the “unwatch” button and then click “delete thread”

**Comment by AshleyPikachu** [comment]
In the meantime Ive removed all content from the post expect for its name, its also not publicly viewable anymore I noticed.

**Comment by AshleyPikachu** [comment]
I see edit thread but no delete option.

**Comment by AustinXII** [comment]
@TheImmortal

**Comment by TheImmortal** [comment]
It has been deleted.

**Comment by AshleyPikachu** [comment]
Yea, sorry hoeenhero took care of it for me. ^^, anyways everything is all cleaned up now,

**Comment by AshleyPikachu** [comment]
Hi, I figure I give you guys an update on our total progress since its almost march.
1. Me and Lycanium Z finished the type chart file 100% today, It will work for Digimon Showdown and the Pokemon X Digimon mode.
2. C7 is working on the Dex Code
3. Alfastorm is coding moves
4. Im choosing the remaining movepools for the digimon
5. After that one of us has to paste in the moveset data.
Then we should be done. We are trying to shoot for March 18th to be done by.

**Comment by AshleyPikachu** [comment]
@Zarel hi, we are working on doing some test to make sure our attacks and statuses are working as coded would it be possible to upload our digimon sprites now so that they are not blocks, the sounds and rest of the assets could be put on later.

**Comment by AustinXII** [comment]
That spoils this project again though?? Uploading the sprites now doesn’t do anything besides make your testing look better unless I’m missing something

**Comment by AshleyPikachu** [comment]
Yea pretty much, thats a good point @AustinXII i just figure it makes it easier to focus on it while testing. No worries thanks anyways.

**Comment by AshleyPikachu** [comment]
Hi, I was @Zarel would it be possible during april fools day to have on the news section some kind of mention that it will be on wavelength all year long, that or we could be it in the battle box html code for the day and alter it later.

**Comment by AshleyPikachu** [comment]
@AustinXII I thought of a compermise on the sprite thing, what if I just make a dot Pikachu in the same style name it Pika??? and we use it as a temp file that way we can make sure scaling is fine.

**Comment by AshleyPikachu** [comment]
Final things to do before the project is finished
Me and Lycanium Z finished the type chart file 100% today, It will work for Digimon Showdown and the Pokemon X Digimon mode.

Attacks all need to be tested to avoid glitches
Animations need to be coded.

That being said @Zarel one status is giving us a really hard time and we could use some help with it.
Its called "Bug", it interacts with 3 abilities Vaccine, Data and Virus. The way it works is type triangle with the 3 abilities. 

Vaccine > Virus
Virus > Data
Data > Vaccine

What the bug status is supose to do is cause a Vaccine Digimon to become Virus, a Virus Digimon to become data and a Data Digimon to become Vaccine for 3 turns.

We also have 2 other custom status's as well called "Panic" and "Dot" they have not been tested yet but there closer to confusion or struggle so they should be easier to fix if they are broken.

Would you like to go over our attack code with us to make sure nothings broken on launch day?
Heres the most recent build https://drive.google.com/file/d/1OS8fkSZdhv3tdzrl5sZg0tLh4ix1D5uJ/view

**Comment by AustinXII** [comment]
I’m not too familiar with all of the PS codebase but don’t you need a onHit?

**Comment by AshleyPikachu** [comment]
the bug error has been fixed thanks to volco were testing stuff now as i write this

**Comment by AshleyPikachu** [comment]
@Zarel everything is done but testing for glitches, we are on schedule :)
When can we have the sprites, attack descriptions and sounds uploaded to the server?

On a side note my staff and I were wondering after the project goes live would it be possible for us to have custom avatars for all the work?

**Comment by AshleyPikachu** [comment]
@AustinXII tested the sprites with inspect element, it went great :)
https://i.imgur.com/tgv8Q3U.png

**Comment by AustinXII** [comment]
Yay! Told ya that’d work better for what you wanted to do :)

**Pull request closed without merging**